.ddev/commands/host/install

```diff
@@ -34,7 +34,7 @@ ddev mysql -uroot -proot <<-EOQ
   GRANT ALL ON ${APP_DB}_test.* TO "${APP_DB}"@'%' IDENTIFIED BY "${APP_DB}";
 EOQ
 
-# generate DB schema
+# generate DB schema for test environment
 ddev exec bin/console doctrine:schema:update --env=test --force --no-interaction
```

Comment:
Yes, we could do that. Most probably you need to execute 
`ddev exec bin/console doctrine:schema:drop --full-database --force --no-interaction --env=test`
once, because the migration table isn't available on your local `db_test` database which leads to a conflict. Or we also drop the schema completely as we did in line 23. What do you think?

Reply:
Since it's a test database, I won't mind dropping and recreating it.

Reply:
Ok.